ansible/README.rst

```diff
@@ -0,0 +1,30 @@
+Ansible and Docker OVN Emulation Guide
```

Comment:
What do you think a folder named "ansible-docker"? I will move all changes under this new folder.